### ISSUE TITLE
Badly looking pprint output in docs for some browsers

### ISSUE THREAD

**Issue by skirpichev** [diofant/diofant#744]
Take, for example, [this page](https://diofant.readthedocs.io/en/latest/tutorial/solvers.html).  This is epiphany (default Gnome browser):
![1](https://user-images.githubusercontent.com/2155800/50356603-b189a580-0563-11e9-9be5-7d95c7871bf2.png)
And this is in Firefox (on same system):
![2](https://user-images.githubusercontent.com/2155800/50356614-c108ee80-0563-11e9-8c27-e1a1789649a5.png)

Same issue in SymPy, e.g. [here](https://docs.sympy.org/latest/tutorial/intro.html).  Epiphany's version:
![3](https://user-images.githubusercontent.com/2155800/50356935-24475080-0565-11e9-8f0e-7b0a66daba10.png)
and Firefox's:
![4](https://user-images.githubusercontent.com/2155800/50356939-2b6e5e80-0565-11e9-9be7-53f9f4dbda07.png)

Maybe different monospace fonts were selected?

**Comment by asmeurer** [comment]
It's a font substitution issue. The monospace font that is used for most of the characters doesn't have the ⎟character, so it is replaced from another font. The problem is that it is likely a font that isn't itself monospace, so it doesn't have the right width. Actually I'm not even sure if two different *monospace* fonts will have the same width as each other. I don't know if it can be fixed with CSS. The best fix I know is to install a font that has all those characters, like DejaVu Sans Mono and set your monospace font in your browser to that. Actually a simple partial fix would be to set the CSS in the docs to prefer that font if it is installed.

**Comment by asmeurer** [comment]
I opened an upstream issue for it. https://github.com/sympy/sympy/issues/15700. It has been discussed before, but I don't think there was an issue for it.

**Comment by skirpichev** [comment]
> It's a font substitution issue. The monospace font that is used for most of the characters doesn't have the ⎟character, so it is replaced from another font.

Hmm, @asmeurer, I see two issues here.

First one: different browsers may select different fonts on the same system (in case, that is important, when page's CSS may override system defaults).  Apparently, the "good" fonts are on the system: epiphany shows docs well.  Maybe it's even a firefox issue: e.g. if I disallow pages to choose fonts and set DejaVu Sans Mono in preferences for monospace - I still see ugly rendered dsolve output.  Element Inspector, surprisingly, still shows several fonts for `<span class="go">f(x) = ℯ ⋅⎜C₂ + ─⎟ + ℯ  ⋅C₁</span>` element.

Second problem: [bad pdf output](https://github.com/diofant/diofant/issues/744#issuecomment-450215143).  I'm not sure that it has same reason.

**Comment by skirpichev** [comment]
Ok, first problem gone if I select "DejaVu Sans" for Serif font in browser preferences.  Or, alternatively, with following CSS:
```css
.rst-content div[class^="highlight"] pre {
    font-family: "DejaVu Sans Mono", "DejaVu Sans", "monospace";
}
```

**Comment by skirpichev** [comment]
In fact, problem seems to be connected with `ℯ` symbol, `exp(1)`, not with `⎟`.  Maybe it does make sense to use `e` for `Exp1` constant?  This still doesn't fix pdf output.

**Comment by skirpichev** [comment]
Problem with PDF output seems to be less severe (and I have no glue how it may be solved), so I [split it out](#754).
